pygbm/loss.py

```diff
@@ -24,20 +40,30 @@ def _get_threads_chunks(total_size):
 @njit(fastmath=True)
 def _expit(x):
     # custom sigmoid because we cannot use that of scipy with numba
+    # TODO make it numerically stable (if needed?)
     return 1 / (1 + np.exp(-x))
 
 
 class Loss(ABC):
 
+    def __init__(self, n_trees_per_iteration=1):
+        self.n_trees_per_iteration_ = n_trees_per_iteration
```

Comment:
The `Loss` objective function is not an estimator, so the `n_trees_per_iteration` attribute does not have to be suffixed with an underscore. The underscore is used distinguish fitted parameters from constructor hyperparameter in scikit-learn estimators.